.changeset/brown-walls-thank.md

```diff
@@ -0,0 +1,5 @@
+---
+"frames.js": patch
```

Comment:
this should be a minor bump right? It is not a bug fix.

Reply:
no breaking changes though

Reply:
yep but I think there is no need to deal with "breaking changes" since minor is not used for breaking changes but extensions to an API. Breaking change would be major bump.